vpr/src/base/read_options.cpp

```diff
@@ -1873,6 +1877,21 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
             "Its range equals to [0., 1.].")
         .default_value("0.7")
         .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_constraint_expand, "--place_constraint_expand")
+        .help(
+            "The value used to decide how much to expand the floorplan constraint region when writing"
+            "a floorplan constraint XML file. If it is zero, the block stays locked in its place. If it is"
+            "greater than zero the constraint region expands by the specified value in each direction.")
+        .default_value("0")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_constraint_subtile, "--place_constraint_subtile")
+        .help(
+            "The value used to specify a subtile constraints when writing a floorplan constraints XML file.")
```

Comment:
should be a boolean option (on or off) otherwise you will have illegal placements (because some blocks can be at the same x, y but diff subtile)

Reply:
if it is off, don't print subtile, if it is on, you print what is in current placement